changelog.md

```diff
@@ -15,6 +15,7 @@ Upcoming release (2022)
     - Confidence intervals added for regression parameters
     - Population plot added with confidence band of regression line
     - :warning: Henze-Zirkler test for assumption of multivariate normality
+    - :warning: White's test and Koenker's test for assumption of homoscedasticity
```

Comment:
```suggestion
    - White's test and Koenker's test for assumption of homoscedasticity
```
Because the new test only adds some new warnings, but will not influence other stats that have been calculated in previous versions, there's no need for the warning sign here